src/System.Management.Automation/FormatAndOutput/common/BaseOutputtingCommand.cs

```diff
@@ -743,6 +743,34 @@ private void ProcessCachedGroupOnWide(WideViewHeaderInfo wvhi, List<PacketInfoDa
             _formattingHint = hint;
         }
 
+        static private int _failedToReadConsoleWidth = 0;
+
+        static private int GetConsoleWindowWidth(int columnNumber)
+        {
+            if (columnNumber == int.MaxValue)
+            {
+                // Tables and Wides need to use spaces for padding to maintain table look even if console window is resized.
+                // For all other output, we use int.MaxValue if the user didn't explicitly specify a width.
+                // If we detect that int.MaxValue is used, first we try to get the current console window width.
+                // However, if we can't read that (for example, implicit remoting has no console window), we default
+                // to something reasonable: 120 columns.
+                if (_failedToReadConsoleWidth > 0)
```

Comment:
I believe bool type is more suitable.
I'd rename this variable to reflect that a console is missing from the remote session.

Reply:
Something like `bool _noConsole`